exercises/space-age/test/test_space-age.c

```diff
@@ -0,0 +1,58 @@
+#include "vendor/unity.h"
+#include "../src/space-age.h"
+
+
+void test_convert_Earth_Year(void)
+{
+   TEST_ASSERT_EQUAL(1, convert_Earth_Age(SECONDS_ON_EARTH_YEAR));
+}
+
+void test_convert_Mercury_Year(void)
+{
+   TEST_ASSERT_EQUAL(0.2408467, convert_Mercury_Age(SECONDS_ON_EARTH_YEAR));
```

Comment:
Unity includes an assertions meant for testing float equality (`TEST_ASSERT_EQUAL_FLOAT`). This should be used to the many complexities of comparing floating point numbers. You can read more about why this is important here: https://randomascii.wordpress.com/2012/02/25/comparing-floating-point-numbers-2012-edition/.